.github/workflows/reusable-dev-deployment.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         id: job-url
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          job_name: Upgrade or install deployment (${{ inputs.name }}, ${{ inputs.sha }}, ${{ inputs.pr_number }}) / Deploy to Kubernetes
+          job_name: Upgrade or install deployment / Deploy to Kubernetes
```

Comment:
Why is this info not available anymore?

Reply:
It is available, but not used anymore for the job_name